tests/test_curves.py

```diff
@@ -205,6 +207,20 @@ def test_to_SplineXYZCurve(self):
             atol=1e-1,
         )
 
+    @pytest.mark.unit
+    def test_from_input_file(self):
+        """Test getting a curve from axis guess in input file."""
+        path = "tests/inputs/input.QSC_r2_5.5_desc"
+
+        curve1 = FourierRZCurve.from_input_file(path)
+        curve2 = Equilibrium(**InputReader(path).inputs[0]).axis
+        curve1.change_resolution(curve2.N)
+
+        np.testing.assert_allclose(curve1.R_n, curve2.R_n)
+        np.testing.assert_allclose(curve1.Z_n, curve2.Z_n)
+        np.testing.assert_allclose(curve1.NFP, curve2.NFP)
+        np.testing.assert_allclose(curve1.sym, curve2.sym)
+
```

Comment:
You can add another test in this function that reads these variables from a vmec input file, say, input.QSC_r2_5.5_vmec. That would increase the coverage.

Reply:
Something like this works and passes the tests.

```      
         path = "tests/inputs/input.QSC_r2_5.5_vmec"                                           
                                                                                                
          with pytest.warns(UserWarning):                                                       
              curve3 = FourierRZCurve.from_input_file(path)                                     
              curve4 = Equilibrium(**InputReader(path).inputs[0]).axis                          
          curve3.change_resolution(curve2.N)                                                    
                                                                                                
          np.testing.assert_allclose(curve3.R_n, curve4.R_n)                                    
          np.testing.assert_allclose(curve3.Z_n, curve4.Z_n)                                    
          np.testing.assert_allclose(curve3.NFP, curve4.NFP)                                    
          np.testing.assert_allclose(curve3.sym, curve4.sym)  
```